presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java

```diff
@@ -442,22 +442,35 @@ public void testJsonToMap()
     public void testElementAt()
             throws Exception
     {
+        // empty map
```

Comment:
One is removed:

```
assertFunction("element_at(MAP(ARRAY [1], ARRAY [1]), 2)", INTEGER, null);
```

But I have these 5:
```
// missing key
assertFunction("element_at(MAP(ARRAY [1], ARRAY [1.0]), 2)", DOUBLE, null);
assertFunction("element_at(MAP(ARRAY [1.0], ARRAY ['a']), 2.0)", createVarcharType(1), null);
assertFunction("element_at(MAP(ARRAY ['a'], ARRAY [true]), 'b')", BOOLEAN, null);
assertFunction("element_at(MAP(ARRAY [true], ARRAY [ARRAY [1]]), false)", new ArrayType(INTEGER), null);
assertFunction("element_at(MAP(ARRAY [ARRAY [1]], ARRAY [1]), ARRAY [2])", INTEGER, null);
```

The rest may be re-formatted, but are all still there.